webapp/channels/src/components/admin_console/blockable_link/blockable_link.tsx

```diff
@@ -24,29 +24,27 @@ type Props = {
     className?: string;
     onClick?: (e: React.MouseEvent) => void;
 };
-export default class BlockableLink extends React.PureComponent<Props> {
-    private handleClick = (e: React.MouseEvent) => {
-        if (this.props.onClick) {
-            this.props.onClick(e);
+
+const BlockableLink = ({blocked, actions, onClick, to, ...restProps}: Props) => {
+    const handleClick = (e: React.MouseEvent) => {
```

Comment:
Since this is passed to a child component, let's use a callback (`useCallback`) to make sure we are not passing a new function on every render.

Reply:
Resolved. 🙌